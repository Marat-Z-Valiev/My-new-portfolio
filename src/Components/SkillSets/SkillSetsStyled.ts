import styled from "styled-components";

const SkillSetsStyled = styled.div`
	padding: 30px;

	#skillSet {
		display: flex;
		flex-direction: column;
		justify-items: center;
	}

	h1 {
		text-align: center;
		color: #f7fdfade;
	}

	.skills {
		display: grid;
		grid-template-columns: repeat(3, 1fr);
		grid-gap: 15px;
		justify-items: center;

		i {
			font-size: 80px;
		}
		img {
			width: 80px;
			height: 80px;
		}
	}
`;

export default SkillSetsStyled;
