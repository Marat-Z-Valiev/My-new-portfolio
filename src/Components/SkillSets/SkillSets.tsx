import React from "react";
import SkillSetsStyled from "./SkillSetsStyled";

const SkillSets: React.FC = () => {
	return (
		<SkillSetsStyled>
			<div id="skillSet">
				<h1>Skill Sets</h1>
				<div className="skills">
					<i className="devicon-html5-plain-wordmark"></i>
					<i className="devicon-css3-plain-wordmark"></i>
					<i className="devicon-javascript-plain"></i>
					<i className="devicon-java-plain-wordmark"></i>
					<i className="devicon-react-original-wordmark"></i>
					<i className="devicon-git-plain-wordmark"></i>
					<i className="devicon-typescript-plain"></i>
					<img src="https://i.ibb.co/XDzR2Lc/responsive-design.png" alt="" />
					<img src="https://i.ibb.co/27b0Qt1/api.png" alt="" />
				</div>
			</div>
		</SkillSetsStyled>
	);
};

export default SkillSets;
