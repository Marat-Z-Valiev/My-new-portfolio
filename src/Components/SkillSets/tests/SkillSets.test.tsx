import React from 'react';
import SkillSets from "../SkillSets";
import { render } from "@testing-library/react";

describe("SkillSets", () => {
  it("should render correctly", () => {
    render(<SkillSets />);
  });
});
