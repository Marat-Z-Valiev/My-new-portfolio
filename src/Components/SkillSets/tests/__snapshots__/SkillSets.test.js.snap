// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkillSets should render correctly 1`] = `
<SkillSetsStyled>
  <div
    id="skillSet"
  >
    <h1>
      Skill Sets
    </h1>
    <div
      className="skills"
    >
      <i
        className="devicon-html5-plain-wordmark"
      />
      <i
        className="devicon-css3-plain-wordmark"
      />
      <i
        className="devicon-javascript-plain"
      />
      <i
        className="devicon-java-plain-wordmark"
      />
      <i
        className="devicon-react-original-wordmark"
      />
      <i
        className="devicon-git-plain-wordmark"
      />
      <i
        className="devicon-typescript-plain"
      />
      <img
        alt=""
        src="https://i.ibb.co/XDzR2Lc/responsive-design.png"
      />
      <img
        alt=""
        src="https://i.ibb.co/27b0Qt1/api.png"
      />
    </div>
  </div>
</SkillSetsStyled>
`;
